tests/functional/Codeception/Docker.php

```diff
@@ -157,32 +159,56 @@ public function cleanUpEnvironment(): bool
             ->stopOnFail();
 
         $this->output = $result->getMessage();
+
         return $result->wasSuccessful();
     }
 
     /**
      * Clones magento cloud template from git
      *
      * @param string|null $version
+     * @param string|null $edition
      * @return bool
-     * @throws \Robo\Exception\TaskException
+     *
+     * @throws TaskException
      */
-    public function cloneTemplate(string $version = null): bool
+    public function cloneTemplate(string $version = null, string $edition = null): bool
     {
-        $gitTask = $this->taskGitStack()
+        $tasks = [];
+        $tasks[] = $this->taskGitStack()
             ->exec('git init')
             ->exec(sprintf('git remote add origin %s', $this->_getConfig('repo_url')))
             ->exec('git fetch')
-            ->checkout($version ?: $this->_getConfig('repo_branch'));
+            ->checkout($version ?: $this->_getConfig('repo_branch'))
+            ->getCommand();
+
+        if ($edition === 'CE') {
+            $tasks[] = $this->taskComposerRemove('composer')
```

Comment:
You are removing `metapackage` and add only the `magento/product-community-edition` package, but metapackage contains more dependencies.
Do we need "magento/module-paypal-on-boarding" and "fastly/magento2" for CE version?

Reply:
We're not testing those extensions on Docker and it seems like overhead to install them explicitly.